index.html

```diff
@@ -123,7 +123,7 @@ <h1 class="headline-lg hero__title" data-reveal="left">
                 <img src="./assets/images/icon-1.png" width="71" height="71" loading="lazy" alt="icon" />
```

Comment:
Interesting size choice..
Make sure the images you're serving are twice as big, so 142x142px in this case. After the resize, compress the PNGs at https://tinypng.com/